### DISCUSSION TITLE
HDMI Output Doesn't Work

### DISCUSSION THREAD

**Discussion by jtitley** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#16] [Q&A] [answered]
Not sure if it's just me, or what.

When I plugin HDMI with DVI adapter nothing happens. When I plugin straight HDMI, the laptop screen will go blank, but the monitor will be fine. If I try to unplug, then nothing. Also, screen extension doesn't work, I guess because the screen is blank.

I tried to look at Hackintool, and it seems it defines the ports as DP not HDMI, but I'm not familiar with the tool to know if this is correct or not. I can say the patch output doesn't match what is in this config.plist

Any troubleshooting advice? Anything I can provide that can give a clue.

Otherwise, everything else works. What outstanding work!

I'm running the latest Big Sur which is 11.1. My hardware profile is identical to you. BIOS 1.15.4 if that helps.

**Comment by adityabakare** [comment, marked as answer]
I have an idea about this, but unfortunately, I didn't find any supporting guide to fix this. It happens with some users. 

The workaround I found for this is to plug the HDMI after the OS fully boots up. Once you plug in, the internal monitor will show nothing, but the external one will have display. Then, put your OS to sleep. Once the external screen goes off too, unplug the HDMI and wake the laptop from sleep. You'll have your internal display back running. And now when you plug the HDMI again, both the monitors should have display.

I've converted this to a discussion for others to confirm this.

**Reply by Aristocrat-jt** [reply]
I just saw this after trying your dvi patch. Will try this before I blow the machine away.

**Reply by adityabakare** [reply]
Actually, I will need to revert one of the framebuffer patches, because apparently HDMI audio seems to be broken. It may/may not affect the DVI problem, but you can still try. I'll do it once Acidanthera releases new OpenCore version.

**Comment by AlexGao1213** [comment]
You can try update your VGA driver or BIOS, If you are normal user ,**windows update** is the best choice.  You should reinstall your os or get help form vender when still happen.

**Comment by Aristocrat-jt** [comment]
The BIOS was updated to the latest version. With other hackintoshs, sometimes the BIOS version is critical. I can try downgrading to a known version that works for people if they can let me know what version they are using. 

Not sure how Windows Update will help here, but for the record, it is dual boot, and the Windows 10 is fully updated. In Windows, the monitor works.
